cairo/ethereum/cancun/state.cairo

```diff
@@ -298,3 +299,62 @@ func set_storage{poseidon_ptr: PoseidonBuiltin*, state: State}(
     );
     return ();
 }
+
+func get_transient_storage{poseidon_ptr: PoseidonBuiltin*, transient_storage: TransientStorage}(
+    address: Address, key: Bytes32
+) -> U256 {
+    alloc_locals;
+    let fp_and_pc = get_fp_and_pc();
+    local __fp__: felt* = fp_and_pc.fp_val;
+
+    let transient_storage_tries_dict_ptr = cast(
+        transient_storage.value._tries.value.dict_ptr, DictAccess*
+    );
+    with poseidon_ptr {
```

Comment:
```suggestion
    with poseidon_ptr {
```

no need to wrap win the with poseidon_ptr as it's already an implicit arg with the correct name

Reply:
done